universal-login-wallet/test/integration/pages/DashboardPage.ts

```diff
@@ -10,8 +10,16 @@ export default class DashboardPage {
     this.wrapper.update();
   }
 
-  disconnect() {
-    this.wrapper.find('.sign-out-btn').simulate('click');
+  areDeleteAccountInputsValid() {
```

Comment:
Is this used anywhere? Besides, if it checks if valid, should be `! this.wrapper.exists('delete-account-input-error')` ?

Reply:
👍